adage/genes/models.py

```diff
@@ -0,0 +1,143 @@
+import re
+
+from django.db import models
+from django.core.exceptions import FieldError
+from organisms.models import Organism
+
+# Regular expressions to be matched in the methods of 'Gene'.
+nonalpha = re.compile(r'[^a-zA-Z0-9]')
+num = re.compile(r'[0-9]')
```

Comment:
I think one reason why these two variables were not included in `Gen` class definition is because it was a class of database models, and adding them in would make them into table fields in the database. 

Actually both variables seem to be used only once by `wall_of_name` method, so we can probably define them directly in that method. Let me check the other modules to make sure they are not referred anywhere else.

Reply:
What did you conclude about these?

Reply:
@mhuyck: I am using issue #2 since `genes` needs further migration.

Reply:
Oh, I missed that these would be covered by fully migrating `django-genes`. That makes sense!